modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts

```diff
@@ -98,3 +101,25 @@ test('LPD-15423 check WebDAV password is generated', async ({
 
 	await expect(editUserPage.webDAVPasswordLabel).toBeVisible();
 });
+
+test('LPD-28908 update user information', async ({
+	apiHelpers,
+	editUserPage,
+	page,
+	usersAndOrganizationsPage
+}) => {
+
+	const user = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	await page.goto('/');
+
+	await usersAndOrganizationsPage.goToUsers();
+	await (await usersAndOrganizationsPage.usersTableRowLink(user.alternateName)).click();
+	const randomNumber = getRandomInt();
```

Comment:
don't save it in a variable but inline it where used because we don't need to have the two fields with the same value (we will remove the user after the test)